bundles/org.openhab.binding.linky/src/main/java/org/openhab/binding/linky/internal/LinkyHandlerFactory.java

```diff
@@ -69,10 +68,9 @@ public boolean supportsThingType(ThingTypeUID thingTypeUID) {
     protected @Nullable ThingHandler createHandler(Thing thing) {
         ThingTypeUID thingTypeUID = thing.getThingTypeUID();
 
-        if (supportsThingType(thingTypeUID)) {
-            return new LinkyHandler(thing, localeProvider, gson, httpClient);
-        }
-
-        return null;
+        return supportsThingType(thingTypeUID)
+                ? new LinkyHandler(thing, localeProvider, gson,
+                        httpClientFactory.createHttpClient(thingTypeUID.getAsString().replace(":", "_")))
```

Comment:
Why do you need a new client for each thing?

Reply:
Because the thing may start or stop the client.

Reply:
But why not having the client setup, started and stopped in the thing handler factory? 
There is nothing depending on thing?

Reply:
Why would the thing handler factory takes care of that ? It does not know is a given thing is activated or not.

Reply:
It would just allow to create only one HTTP client for the binding and not one for each thing. 
But this is not very important as 99,9℅ of users will define only one thing, so it makes no difference in practice.

Reply:
@J-N-K : with the explanation, is it ok for you?

Reply:
Of course hthe shared client should never be stopped, so this if of course a good reason. But why is it necessary to start/stop the client?